package.json

```diff
@@ -7,5 +7,8 @@
         "sample",
         "website",
         "development"
-    ]
+    ],
+    "overrides": {
+        "react": "^19.0.0"
+    }
```

Comment:
This is needed since `react-helmet-async` doesn't officially support React 19 yet. But it seems to work fine. I'm a little concerned that it's a dead project, but I'll probably submit a MR to it anyway.

See https://github.com/staylor/react-helmet-async/issues/239